routes/config.json

```diff
@@ -84,7 +84,17 @@
     "routers": ["util", "populate", "common"],
     "params": {
       "populate": {
-        "references": ["_categories", "_brand", "_images"]
+        "references": [
+          {
+            "path": "_categories", 
+            "model": "Category" 
+          }, 
+          "_brand", 
```

Comment:
why this one is different?

Reply:
the underscore( _ ) indicates the reference field.